### ISSUE TITLE
Watering scheduler starts watering twice on one pod

### ISSUE THREAD

**Issue by ErhardSiegl** [Gepardec/Hogarama#225]
Watering is started twice at the same time at the same pod:

```
  | 07:24:10,000 INFO  [com.gepardec.hogarama.service.schedulers.WateringScheduler] (EJB default - 2) Check for watering
  | 07:24:10,000 INFO  [com.gepardec.hogarama.service.schedulers.WateringScheduler] (EJB default - 1) Check for watering
  | 07:24:10,001 INFO  [com.gepardec.hogarama.service.schedulers.SensorsScheduler] (EJB default - 6) Load the sensorNames from the database
  | 07:24:10,000 INFO  [com.gepardec.hogarama.service.schedulers.SensorsScheduler] (EJB default - 9) Load the sensorNames from the database
  | 07:24:10,018 INFO  [com.gepardec.hogarama.service.ActorServiceImpl] (EJB default - 2) sendActorMessage: location: Gruenbach, actorName: GruenesBasilikum, duration: 10
  | 07:24:10,019 INFO  [com.gepardec.hogarama.service.ActorServiceImpl] (EJB default - 1) sendActorMessage: location: Gruenbach, actorName: GruenesBasilikum, duration: 10
  | 07:24:10,023 INFO  [com.gepardec.hogarama.mocks.cli.MqttClient] (EJB default - 1) Publising to ssl://broker-amq-mqtt-ssl-57-hogarama.cloud.itandtel.at:443
  | 07:24:10,024 INFO  [com.gepardec.hogarama.mocks.cli.MqttClient] (EJB default - 2) Publising to ssl://broker-amq-mqtt-ssl-57-hogarama.cloud.itandtel.at:443
  | 07:24:10,107 INFO  [com.gepardec.hogarama.mocks.cli.MqttClient] (EJB default - 1) Published 1 of 1
  | 07:24:10,109 INFO  [com.gepardec.hogarama.mocks.cli.MqttClient] (EJB default - 2) Published 1 of 1
```
This results to two watering events at the client.

**Comment by ErhardSiegl** [comment]
Maybe fix it together with #221

**Comment by ErhardSiegl** [comment]
The WateringScheduler is deployed twice:
```
2019-08-14 18:14:45,750 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-6) WFLYEJB0473: JNDI bindings for session bean named 'WateringScheduler' in deployment unit 'deployment "hogajama-frontend-0.0.1-SNAPSHOT.war"' are as follows:

    java:global/hogajama-frontend-0.0.1-SNAPSHOT/WateringScheduler!com.gepardec.hogarama.service.schedulers.WateringScheduler
    java:app/hogajama-frontend-0.0.1-SNAPSHOT/WateringScheduler!com.gepardec.hogarama.service.schedulers.WateringScheduler
    java:module/WateringScheduler!com.gepardec.hogarama.service.schedulers.WateringScheduler
    java:global/hogajama-frontend-0.0.1-SNAPSHOT/WateringScheduler
    java:app/hogajama-frontend-0.0.1-SNAPSHOT/WateringScheduler
    java:module/WateringScheduler

2019-08-14 18:14:45,750 INFO  [org.jboss.as.ejb3.deployment] (MSC service thread 1-2) WFLYEJB0473: JNDI bindings for session bean named 'WateringScheduler' in deployment unit 'deployment "hogajama-rs-0.0.1-SNAPSHOT.war"' are as follows:

    java:global/hogajama-rs-0.0.1-SNAPSHOT/WateringScheduler!com.gepardec.hogarama.service.schedulers.WateringScheduler
    java:app/hogajama-rs-0.0.1-SNAPSHOT/WateringScheduler!com.gepardec.hogarama.service.schedulers.WateringScheduler
    java:module/WateringScheduler!com.gepardec.hogarama.service.schedulers.WateringScheduler
    java:global/hogajama-rs-0.0.1-SNAPSHOT/WateringScheduler
    java:app/hogajama-rs-0.0.1-SNAPSHOT/WateringScheduler
    java:module/WateringScheduler
```

**Comment by fhochleitner** [comment]
Fixed with commit a5b582d
